doc/code/interfaces/numpy.rst

```diff
@@ -42,7 +42,7 @@ but this can also be specified explicitly by passing the ``interface='numpy'`` k
         qml.RY(phi[1], wires=1)
         qml.CNOT(wires=[0, 1])
         qml.PhaseShift(theta, wires=0)
-        return qml.expval.PauliZ(0), qml.expval.Hadamard(1)
+        return qml.expval(qml.PauliZ(0)), qml.expval(qml.Hadamard(1))
```

Comment:
Just wanted to get your thoughts on the following pattern:
```python
from pennylane import expval, var, sample
...

    return expval(qml.PauliZ(0)), var(qml.PauliX(1))
```

This would only be a convention change for the documentation, but could be clearer to read/parse for users.

Reply:
Don't think this is important for this PR though, can always think about this later.

Reply:
Yeah, I realized that we needed to add something. I was also thinking of the classic `from pennylane.ops import *` followed by `qml.expval(PauliZ(0))`. In the end, I decided with the pattern in the PR simply due to the fact that we have so many code stubs everywhere, and I wanted everything to run with only the basic `import pennylane as qml` (which all users would know to do)

Reply:
My preference would be against `pennylane.ops import *`, just because it could lead to namespace conflicts for users and lead to hard to debug issues. But `from pennylane import expval`, or even `from pennylane import PauliX, PauliZ` could be a middle ground solution.